README.md

```diff
@@ -39,26 +39,56 @@ To run the internet_identity canisters, proceed as follows after cloning the rep
 ```bash
 npm install
 dfx start [--clean] [--background]
+```
+
+In a different terminal, run the following command to install the Internet Identity canister:
+
+```bash
 II_ENV=development dfx deploy --no-wallet --argument '(null)'
 ```
 
 Then the canister can be used as
 
 ```bash
-dfx canister call internet_identity register '(123, "test", vec {1; 2; 3}, null)'
+$ dfx canister call internet_identity init_salt
+()
+$ echo $?
+0
+```
+
+See `dfx canister call --help` and [the documentation](https://sdk.dfinity.org/docs/developers-guide/cli-reference/dfx-canister.html#_examples) for more information.
+
+The `dfx` executable can proxy queries to the canister. To view it, run the following:
+
+```bash
+open "http://localhost:8000?canisterId=$(dfx canister id internet_identity)"
 ```
 
-To open the front-end, you can run the following and open the URL.
+To serve the frontend locally via webpack (recommended during development), run
+the following:
 
 ```bash
-echo "http://localhost:8000?canisterId=$(dfx canister id internet_identity)"
+CANISTER_ID=$(dfx canister id internet_identity) npm start
+open "http://localhost:8080"
 ```
 
+Finally, to test workflows like authentication from a client application, you start the sample app:
```

Comment:
I think this section should go under `Contributing to the frontend`, where you have the TODO right now

Reply:
Agreed